### DISCUSSION TITLE
Searching releases should display the matched text

### DISCUSSION THREAD

**Discussion by idbrii** [community/community#6093] [Other Feature Feedback, Questions, & Ideas] [answered]
If I search for text that's in the description of a release and it's not at the top, then I get a bunch of results that look identical:

![none of these are signed](https://user-images.githubusercontent.com/43559/136452663-09fd53b0-24ce-4c95-9229-0c74318c2540.png)

The results should either show the full content of each release or include the matched strings. I think full content is preferable so because different users will want different amounts of context.

----

A specific example: some of [vim/vim-win32-installer's releases](https://github.com/vim/vim-win32-installer/releases) contain a "🔒 Signed Files" heading. Currently, [searching for Signed Files](https://github.com/vim/vim-win32-installer/releases?q=Signed+Files) returns lots of false positives because it matches "Unsigned Files". Since it doesn't expand, you can't sort the false positives from desired matches and you still can't find what you're looking for.

**Comment by MylesBorins** [comment, marked as answer]
We are planning to do exactly this, expand content and highlight search terms.  Thanks for the suggestion

**Reply by MylesBorins** [reply]
as an FYI we made major improvements to search this week and are kicking off work on highlighting next week

**Reply by idbrii** [reply]
The expanded search results are great!
